apps/backend/src/database/migrations/1690503718109-daily-date.ts

```diff
@@ -0,0 +1,42 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migration1690503718109 implements MigrationInterface {
+  name = 'daily-date-migration1690503718109';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "pos_deposit" ADD "file_created_date" TIMESTAMP`
```

Comment:
Need an explanation for this migration.
- If each deposit / transaction is associated with a file, why the repeated column on each?
- Is file_created_date meaningfully different from daily_date?

Reply:
I just thought that it should be included in the file metadata since we're extracting that from the file itself...

Reply:
(I think there was talk of removing the metadata column altogether? I'm just using this for the reporting uploaded date but can remove and refactor to use the daily date instead)